cql/src/main/java/io/stargate/cql/CqlActivator.java

```diff
@@ -99,6 +99,19 @@ private static Config makeConfig() {
       c.rpc_address = listenAddress;
       c.native_transport_port = cqlPort;
 
+      c.native_transport_max_concurrent_connections =
+          Long.getLong("stargate.cql.native_transport_max_concurrent_connections", -1);
```

Comment:
do we want it to be unbounded? Wat value those had before?
(this and `stargate.cql.native_transport_max_concurrent_connections_per_ip"`)

Is it possible to add an Integration Test for it?

Reply:
I guess these options are deprecated now or?

Reply:
It was unbounded in Stargate before this PR, and it's unbounded by default in OSS Cassandra.

Reply:
As the general practice it seems that unbounded is not a good choice? I realize that we do have to worry about backwards compatibility but maybe some high enough but bounded value would be better?